Source/JCC/config.cpp

```diff
@@ -206,6 +873,115 @@ class CfgPatches
+					class Object14 {side=8;vehicle="Land_WoodPile_F";rank="";position[]={5.99512,1.52832,-0.00143862};dir=0;};
+					class Object15 {side=8;vehicle="Land_FoodSacks_01_cargo_brown_F";rank="";position[]={1.36963,-6.50391,-0.00243855};dir=360;};
+					class Object16 {side=8;vehicle="Land_Slum_01_F";rank="";position[]={1.40967,-8.99854,-0.00143862};dir=90.1036;};
+					class Object17 {side=8;vehicle="Land_WoodenTable_02_large_F";rank="";position[]={-7.04639,0.438477,0.0114694};dir=0.00016664;};
+					class Object18 {side=8;vehicle="Land_FoodSacks_01_cargo_brown_F";rank="";position[]={2.979,-6.47559,-0.00243855};dir=360;};
+					class Object19 {side=8;vehicle="Land_ClothesLine_01_short_F";rank="";position[]={-4.625,-5.97656,-0.00143862};dir=0;};
+					class Object20 {side=8;vehicle="Land_PaperBox_01_open_water_F";rank="";position[]={-3.33203,6.8584,-0.000507355};dir=360;};
+					class Object21 {side=8;vehicle="Land_Campfire_F";rank="";position[]={-0.0512695,9.81836,0.0285602};dir=0;};
+					class Object22 {side=8;vehicle="Land_Slum_04_F";rank="";position[]={-7.92188,6.60596,-0.00143862};dir=0;};
+					class Object23 {side=8;vehicle="Box_Syndicate_WpsLaunch_F";rank="";position[]={3.33252,-9.88184,-0.00243902};dir=266.449;};
+					class Object24 {side=8;vehicle="Land_PaperBox_01_small_closed_white_med_F";rank="";position[]={-3.76904,9.86426,-0.00143433};dir=51.6367;};
+					class Object25 {side=8;vehicle="Land_Slum_04_F";rank="";position[]={8.22412,7.00049,-0.00143862};dir=0;};
+					class Object26 {side=8;vehicle="Box_IED_Exp_F";rank="";position[]={1.25879,-10.7876,-0.00243998};dir=301.15;};
+					class Object27 {side=8;vehicle="Box_IED_Exp_F";rank="";position[]={-0.225586,-10.8774,-0.0024395};dir=53.9253;};
+					class Object28 {side=8;vehicle="Land_WoodPile_large_F";rank="";position[]={10.8081,1.71436,0.0217681};dir=88.5004;};
+					class Object29 {side=8;vehicle="Land_ClothesLine_01_full_F";rank="";position[]={9.44824,-5.72412,-0.00143862};dir=0;};
+					class Object30 {side=8;vehicle="Box_Syndicate_Ammo_F";rank="";position[]={2.57813,-10.9834,-0.00243998};dir=309.632;};
+					class Object31 {side=8;vehicle="Land_PaperBox_01_small_closed_white_med_F";rank="";position[]={-4.00537,10.9072,-0.00143862};dir=94.3978;};
+					class Object32 {side=8;vehicle="Box_Syndicate_WpsLaunch_F";rank="";position[]={3.39111,-11.3301,-0.00243902};dir=266.449;};
+					class Object33 {side=8;vehicle="Box_Syndicate_Ammo_F";rank="";position[]={-0.427734,-11.8804,-0.00243902};dir=3.19189e-006;};
+					class Object34 {side=8;vehicle="Box_Syndicate_Ammo_F";rank="";position[]={0.472168,-11.896,-0.0024395};dir=360;};
+					class Object35 {side=8;vehicle="Box_Syndicate_Wps_F";rank="";position[]={1.48828,-11.9165,-0.00243855};dir=360;};
+					class Object36 {side=8;vehicle="Box_Syndicate_Wps_F";rank="";position[]={2.54053,-11.8994,0.0157061};dir=360;};
+				};
+				class syndikatExecution
```

Comment:
What is "Syndikat Execution"?

Reply:
> What is "Syndikat Execution"?

Execution ground for well people who might want to use that, i just didn't know where to put it under so i put it under syndikat.

Reply:
Okay, but maybe a better name for it? Can you send a picture of what it looks like?

Reply:
> Okay, but maybe a better name for it? Can you send a picture of what it looks like?

Sure give me a min

Reply:
`syndikatExecutionEnclosure`?